src/Components/Weather/WeatherCard.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import constant from 'data/const';
+import styles from './WeatherCard.sass';
+
+class WeatherCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: undefined,
+    };
+  }
+
+  componentDidMount() {
+    const API_URL = `http://api.openweathermap.org/data/2.5/weather?q=${this.props.location}&appid=${constant.WEATHER_API_KEY}&units=metric`;
+    let weatherData;
+
+    fetch(API_URL).then(res => res.json()).then(json => {
+      weatherData = {
+        temperature: json.main.temp,
+        city: json.name,
+        country: json.sys.country,
+        time: new Date(json.dt).toLocaleString(),
+        humidity: json.main.humidity,
+        description: json.weather[0].description,
+        iconName: json.weather[0].icon,
+        error: '',
+      };
+      this.setState({ data: weatherData });
+    });
+  }
+
+  render() {
+    const wData = this.state.data;
+    const iconUrl = 'http://openweathermap.org/img/w/' + (wData ? wData.iconName : '') + '.png';
+
+    return (
+      <div className={styles.weatherCard}>
+        { wData
+          ? (
+            <div className={styles.weatherCardContent}>
+              <header className={styles.weatherCardHeader}>
+                <h2 className={styles.weatherCardLocation}>
+                  {wData.city}
+                  ,
+                  &nbsp;
+                  {wData.country}
```

Comment:
This is what linter says - move all to the next line.. I don't like it, how to avoid this?